src/libwalletqt/Wallet.cpp

```diff
@@ -457,6 +515,34 @@ AddressBookModel *Wallet::addressBookModel() const
     return m_addressBookModel;
 }
 
+Subaddress *Wallet::subaddress() const
+{
+    return m_subaddress;
+}
+
+SubaddressModel *Wallet::subaddressModel() const
+{
```

Comment:
Given this does not return a const SuaddressModel, and it const casts this below, should this const be removed ?

Reply:
I think this member function was made `const` and the `m_subaddressAccountModel` member data `mutable` because `m_subaddressAccountModel` gets assigned a newly instantiated object when it's pointed to `nullptr` (i.e., *NOT* because it returns a non-const `SubaddressModel*` -- it's perfectly OK for a const member function to return a (copy of ) member data which is a pointer to a non-const object). But anyway I made those functions (`subaddress(), subaddressModel(), subaddressAccount(), subaddressAccountModel()`) non-const and removed `mutable`, as I don't see any problem in doing so.